jbt-it/neomdb
# #130 server implement trainee admission

- Added admission route for trainees.
- Changed member status to its own type

**How to test**:
1. Run admission tests
2. Make sure, that one member in the database is a trainee
3. Use swagger (localhost:3030/api/docs) to test the admission route by providing the id of the trainee
4. Check if the member is now active and the aktivSeit attribute also changed to the current date
